game.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'item'
+
+class Game < Item
+  def initialize(date)
+    super
+    @multiplayer = false
+    @last_played_at = date
+    @list_of_games = []
+  end
```

Comment:
You made a good job, but I think you need to add the parameter in the `initialize()` like this:

```ruby
def initialize(date, publish_date)
    super(publish_date)
    @multiplayer = false
    @last_played_at = date
    @list_of_games = []
  end
```